README.md

```diff
@@ -69,15 +51,29 @@ Once you have built the command line tool, you will find two executable files fo
 ```
 > .\Microsoft.Health.Fhir.Anonymizer.<version>.CommandLineTool.exe -i myInputFolder -o myOutputFolder
 ```
-See the [reference](#the-command-line-tool) section for usage details of the command line tool.
 
-# Tutorials
+The command-line tool can be used to anonymize a folder containing FHIR resource files. Here are the parameters that the tool accepts:
 
-## Anonymize FHIR data using Azure Data Factory
+| Option | Name | Optionality | Default | Description |
+| ----- | ----- | ----- |----- |----- |
+| -i | inputFolder | Required | | Folder to locate input resource files. |
+| -o | outputFolder | Required | |  Folder to save anonymized resource files. |
+| -c | configFile | Optional |configuration-sample.json | Anonymizer configuration file path. It reads the default file from the current directory. |
+| -b | bulkData | Optional| false | Resource file is in bulk data format (.ndjson). |
+| -r | recursive | Optional | false | Process resource files in input folder recursively. |
+| -v | verbose | Optional | false | Provide additional details during processing. |
+| -s | skip | Optional | false | Skip files that are already present in the destination folder. |
+| --validateInput | validateInput | Optional | false | Validate input resources against structure, cardinality and most value domains in FHIR specification. Detailed report can be found in verbose log. |
+| --validateOutput | validateOutput | Optional | false | Validate anonymized resources against structure, cardinality and most value domains in FHIR specification. Detailed report can be found in verbose log. |
 
-In this tutorial, you use the Azure PowerShell to create a Data Factory and a pipeline to anonymize FHIR data. The pipeline reads from an Azure blob container, anonymizes it as per the configuration file, and writes the output to another blob container. If you're new to Azure Data Factory, see [Introduction to Azure Data Factory](https://docs.microsoft.com/en-us/azure/data-factory/introduction).
+Example usage to anonymize FHIR resource files in a folder: 
+```
+> .\Microsoft.Health.Fhir.Anonymizer.<version>.CommandLineTool.exe -i myInputFolder -o myOutputFolder
+```
 
-Tutorial steps:
+## Anonymize FHIR data: using Azure Data Factory
```

Comment:
Can we move the details (ADF pipelines, command-line tool options, etc.) to a separate file in `docs\` and just leave some brief introduction in `README.md`? Currently the `README.md` looks a bit long, especially after adding the new contents.